integrations/tensorflow-serving/documentation.yaml

```diff
@@ -0,0 +1,115 @@
+exporter_type: included
+app_name_short: TF Serving
+app_name: TensorFlow Serving
+app_site_name: {{app_name_short}}
+app_site_url: https://github.com/tensorflow/serving
+exporter_name: {{app_name_short}}
+exporter_pkg_name: {{app_name_short}}
+exporter_repo_url: https://www.tensorflow.org/tfx/serving/serving_config#monitoring_configuration
+additional_prereq_info: |
+  {{app_name_short}} exposes Prometheus-format metrics when the `--monitoring_config_file` flag
+  is used to specify a file containing a MonitoringConfig protocol buffer.
+  
+  This is an example:
+    prometheus_config {
+    enable: true,
+    path: "/monitoring/prometheus/metrics"
+  }
+config_mods: |
+  # Copyright 2025 Google LLC
+  #
+  # Licensed under the Apache License, Version 2.0 (the "License");
+  # you may not use this file except in compliance with the License.
+  # You may obtain a copy of the License at
+  #
+  #      http://www.apache.org/licenses/LICENSE-2.0
+  #
+  # Unless required by applicable law or agreed to in writing, software
+  # distributed under the License is distributed on an "AS IS" BASIS,
+  # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  # See the License for the specific language governing permissions and
+  # limitations under the License.
+
+  apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    name: tfserve-deployment
+    labels:
+      app: tfserve-server
+  spec:
+    selector:
+      matchLabels:
+        app: tfserve
+    replicas: 1
+    template:
+      metadata:
+        labels:
+          app: tfserve
+        annotations:
+          gke-gcsfuse/volumes: "true"
+      spec:
+        nodeSelector:
+          cloud.google.com/gke-accelerator: "nvidia-l4"
+        containers:
+        - name: tfserve-server
+          image: tensorflow/serving:2.13.1-gpu
+  +       command: [ "tensorflow_model_server", "--model_name=$MODEL_NAME", "--model_base_path=/data/tfserve-model-repository/$MODEL_NAME", "--rest_api_port=8000", "--monitoring_config_file=$PATH/TO/MONITORING/CONFIG" ]
```

Comment:
I'd probably make this `$PATH_TO_MONITORING_CONFIG`, since the slashes are unlikely to be part of an actual envar, and it's not `${PATH}/TO/MONITORING/CONFIG`.

Reply:
Done.